docs/user-guide/liveness/README.md

```diff
@@ -66,11 +66,13 @@ The [http-liveness.yaml](http-liveness.yaml) demonstrates the HTTP check.
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          X-Custom-Header: [ Awesome ]
```

Comment:
That's the actual type for http.Header. It results in the same header being output multiple times, with different values for each occurrence. I guess it might make sense for e.g. chains of X-Forwarded-For? Or for cookies, see http://stackoverflow.com/questions/3241326/set-more-than-one-http-header-with-the-same-name